lib/api/controllers/funnelController.js

```diff
@@ -398,6 +415,32 @@ class FunnelController {
       });
   }
 
+  /**
+   * @param {KuzzleRequest} request
+   * @param {String} prefix
+   * @return {KuzzleRequest}
+   */
+  performDocumentAlias(request, prefix = 'before') {
```

Comment:
Remove the default parameter since you explicitly specify it with each call (which is good)